apps/admin_api/test/admin_api/v1/controllers/blockchain_balance_controller_test.exs

```diff
@@ -318,14 +314,14 @@ defmodule AdminAPI.V1.BlockchainBalanceControllerTest do
         end)
 
       assert length(balances) == 2
-      assert Enum.member?(balances, {"0x0000000000000000000000000000000000000000", 123})
+      assert Enum.member?(balances, {EthBlockchain.eth_address(), 123})
       assert Enum.member?(balances, {"0x0000000000000000000000000000000000000002", 123})
     end
 
     test_with_auths "returns an error when given a non-existing address" do
       response =
         request("/blockchain_wallet.get_balances", %{
-          "address" => "0x00000000000000000000000000000000000000000"
+          "address" => EthBlockchain.eth_address()
```

Comment:
This attribute is for a non-existing wallet address, so I think it's better to be a random address than `EthBlockchain.eth_address()`